design/ceph/object-store-zone.md

```diff
@@ -0,0 +1,183 @@
+# Rook Object Store Zone
+
+## Prerequisites
+
+A rook ceph cluster with an object-store-realm and object-store-zonegroup resource created. 
```

Comment:
what about the case where we didn't create real/zonegroup and just want to use the "defaults" (for backward compatibility current implementation)?

Reply:
In that case, the zone wouldn't get created either.